src/liboslcomp/codegen.cpp

```diff
@@ -98,8 +98,7 @@ OSLCompilerImpl::insert_code (int opnum, const char *opname,
     // the jump addresses of other ops and the param init ranges.
     if (opnum < (int)m_ircode.size()-1) {
         // Adjust jump offsets
-        for (size_t n = 0;  n < m_ircode.size();  ++n) {
-            Opcode &c (m_ircode[n]);
+        for (auto & c : m_ircode) {
```

Comment:
D'oh!  That's what `clang-tidy --fix` did. I tried to fix them all by hand, looks like I missed one or two. I'll fix.